client/src/amplify-ui/cli-feature-flag/feature-flag-summary.tsx

```diff
@@ -0,0 +1,54 @@
+import {Component, Host, h, Prop, Element} from "@stencil/core";
+import {FeatureFlag} from "./feature-flag";
+import {tableStyle, summaryCell} from "./feature-flag.style";
+
+@Component({tag: "amplify-feature-flag-summary", shadow: false})
+export class AmplifyFeatureFlagSummary {
+  @Element() el: HTMLElement;
```

Comment:
I do not see this used anywhere, is it needed?

Reply:
Nope, leftover